### ISSUE TITLE
Twig shouldUnwrapConditional Error

### ISSUE THREAD

**Issue by bymayo** [craftcms/cms#4150]
### Description

I had an issue where `__home__` was being changed to `home` in a slug. So I updated the CMS as I noticed a fix in the changelog (https://github.com/craftcms/cms/blob/develop/CHANGELOG-v3.md#31211---2019-04-04) 

But after doing this, I get the below error whenever I go to the Dashboard or Entries. All other pages seem to be fine.

```
Argument 1 passed to Twig\NodeVisitor\EscaperNodeVisitor::shouldUnwrapConditional() must be an instance of Twig\Node\Expression\ConditionalExpression, instance of Twig\Node\Expression\InlinePrint given, called in /craft-3/vendor/twig/twig/src/NodeVisitor/EscaperNodeVisitor.php on line 110
```

It seems on the dashboard this error is being called from `craft-3/vendor/craftcms/cms/src/controllers/DashboardController.php at line 71 – craft\widgets\Feed::getSettingsHtml()`

### Additional info

- Craft version: 3.1.22
- PHP version: 7.1.22
- Database driver & version: MySQL 5.7
- Plugins & versions: Many!

**Comment by NickThompson1993** [comment]
+1 on this, craft 3.1.19 php 7.3.something MySQL 5.7

Looks like it was caused by the 2.8.0 twig update?

**Comment by bymayo** [comment]
Yeah I've just noticed in Discord that the discussion is around Twig 2.8.0 release. I've downgraded by adding:

`"twig/twig": "2.7.4"`

To my `composer.json`.

Thanks to https://twitter.com/jalendport/status/1118154116289761280

**Comment by andris-sevcenko** [comment]
Fixed by releasing Craft 3.1.23.

It's safe to remove the local twig requirement now.

**Comment by fabpot** [comment]
Twig versions 1.39.1 and 2.8.1 released (with a fix for this issue, sorry about the inconvenience).
